run_kind_ci.sh

```diff
@@ -11,14 +11,15 @@ export PHASES_TO_RUN=("${KIND_CI_PHASES[@]}")
 usage() {
   echo "usage: run_kind_ci.sh [[--project <name>] [--phases <phases>] [--skip-phases <phases>]"
   echo "                       [--num-workers <num>] [--kind-config <conf-fil>] [--kubeconfig <path>]"
-  echo "                       [--kind-node-image <image>] [-h]]"
+  echo "                       [--kind-node-image <image>] [--interfaces-type <interfaces-type>] [-h]]"
   echo ""
   echo "--project              Project to run CI: ${KIND_CI_PROJECTS[*]}"
   echo "                       Required field"
   echo "--phases               Comma separated phases to run, if presented ignores --skip-phases. phases: ${KIND_CI_PHASES[*]}"
   echo "--skip-phases          Comma separated phases, phases: ${KIND_CI_PHASES[*]}"
   echo "--num-workers          Number of worker nodes. DEFAULT: 2 worker"
   echo "--pf-per-worker        How many PFs to switch for each worker node"
+  echo "--interfaces-type      Specify the interfaces type to switch into the kind node. support: "eth", "ib", or "both". Default: eth"
```

Comment:
There is no 'both' usage in this PR

Reply:
basically, if it is not ib or ethernet, it would take all mlnx interfaces discovered. so any word even if it is not both would basically do both.

It was added as a way to change the default value of eth, even if it does not have a specific if statement.